SocialCareCaseViewerApi.Tests/V1/Gateways/Database/GetPersonWithPersonalRelationshipsByPersonIdTests.cs

```diff
@@ -0,0 +1,104 @@
+using System.Linq;
+using FluentAssertions;
+using MongoDB.Driver;
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using SocialCareCaseViewerApi.V1.Gateways;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Gateways.Database
+{
+    [TestFixture]
```

Comment:
Should there be a test for a case where the person with the given id doesn't exist in the database?

Reply:
Definitely! 🤦‍♀️

Reply:
Added `if (personWithRelationships == null) return null;` check alongside a test.